backend/etc/env_template/.env.dev

```
@@ -0,0 +1,17 @@
+PROFILE=
+DB_URL=
+DB_DATABASE=
+DB_USERNAME=
+DB_PASSWORD=
+DB_ROOT_PASSWORD=
+MONGO_HOST=
+MONGO_PORT=
+MONGO_DATABASE=
+MONGO_ROOT_USERNAME=
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The MONGO_ROOT_USERNAME key should go before the PROFILE key

(UnorderedKey)